### ISSUE TITLE
Can't deploy sample for oscar-worker

### ISSUE THREAD

**Issue by alexellis** [grycap/oscar-worker#15]
Hi folks, this is super interesting and a ping-back from: https://github.com/openfaas/faas/issues/657

I'm trying to deploy the sample, but can't get it working?

```
kubectl apply -f yaml/
namespace/oscar created
namespace/oscar-fn created
serviceaccount/oscar-worker-controller created
clusterrole.rbac.authorization.k8s.io/oscar-worker-controller created
clusterrolebinding.rbac.authorization.k8s.io/oscar-worker-controller-global created
Error from server (BadRequest): error when creating "yaml/oscar-worker-dep.yaml": Deployment in version "v1beta1" cannot be handled as a Deployment: v1beta1.Deployment.Spec: v1beta1.DeploymentSpec.Template: v1.PodTemplateSpec.Spec: v1.PodSpec.Containers: []v1.Container: v1.Container.Env: []v1.EnvVar: v1.EnvVar.Value: ReadString: expects " or n, but found 6, error found in #10 byte of ...|,"value":60},{"name"|..., bigger context ...|{"name":"JOB_TTL_SECONDS_AFTER_FINISHED","value":60},{"name":"JOB_BACKOFF_LIMIT","value":3}],"image"|...
```

```
kubectl version
Client Version: version.Info{Major:"1", Minor:"14", GitVersion:"v1.14.3", GitCommit:"5e53fd6bc17c0dec8434817e69b04a25d8ae0ff0", GitTreeState:"clean", BuildDate:"2019-06-06T01:44:30Z", GoVersion:"go1.12.5", Compiler:"gc", Platform:"darwin/amd64"}
Server Version: version.Info{Major:"1", Minor:"14", GitVersion:"v1.14.1", GitCommit:"b7394102d6ef778017f2ca4046abbaa23b88c290", GitTreeState:"clean", BuildDate:"2019-04-08T17:02:58Z", GoVersion:"go1.12.1", Compiler:"gc", Platform:"linux/amd64"}
```

**Comment by srisco** [comment]
Fixed in d850679d2eecce40ebbc3675dc837ccd642ab3e2.

Let me know if you can deploy it now.

**Comment by alexellis** [comment]
![Screenshot 2019-07-01 at 12 59 27](https://user-images.githubusercontent.com/6358735/60434889-18b5a000-9c00-11e9-882a-6f7487eb6c87.png)

👍 it's been fixed, here's my test-run with nodeinfo, but please see also: #16
